apps/storefront/src/pages/Login/index.tsx

```diff
@@ -244,7 +244,10 @@ export default function Login(props: PageProps) {
         } = await b2bLogin({ loginData });
 
         storeDispatch(setB2BToken(token));
-        customerLoginAPI(storefrontLoginToken);
+
+        if (platform === 'bigcommerce') {
+          customerLoginAPI(storefrontLoginToken);
+        }
```

Comment:
shouldn't we move this logic inside `customerLoginAPI` since that api call will only work when platform is `bigcommerce`?

otherwise we'd have to do this if statement if we use this api anywhere else also.

Reply:
You're right, let me update that